File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/AccountInfo.scala
package io.swagger.server.model


/**
 * = AccountInfo =
 *
 * Contains the full account profile including personal info, billing details, OAuth connections, and security settings.
 *
 * @param custid 
 * @param ima 
 * @param data 
 * @param ip 
 * @param oauthproviders 
 * @param oauthconfig 
 * @param oauthadapters 
 * @param limits 
 * @param language 
 * @param countryCurrencies 
 * @param enableLocales 
 * @param enableCurrencies 
 * @param gravatar 
 */
case class AccountInfo (
  custid: Option[String],
  ima: Option[String],
  data: Option[AccountInfoData],
  ip: Option[String],
  oauthproviders: Option[OneOfAccountInfoOauthproviders],
  oauthconfig: Option[AccountInfoOauthConfig],
  oauthadapters: Option[List[String]],
  limits: Option[AccountInfoLimits],
  language: Option[String],
  countryCurrencies: Option[AccountInfoCountryCurrencies],
  enableLocales: Option[Boolean],
  enableCurrencies: Option[Boolean],
  gravatar: Option[String]
)

File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/AccountInfo.js
goog.provide('API.Client.AccountInfo');

/**
 * Contains the full account profile including personal info, billing details, OAuth connections, and security settings.
 * @record
 */
API.Client.AccountInfo = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.AccountInfo.prototype.custid;

/**
 * @type {!string}
 * @export
 */
API.Client.AccountInfo.prototype.ima;

/**
 * @type {!API.Client.AccountInfoData}
 * @export
 */
API.Client.AccountInfo.prototype.data;

/**
 * @type {!string}
 * @export
 */
API.Client.AccountInfo.prototype.ip;

/**
 * @type {!API.Client.AccountInfo_oauthproviders}
 * @export
 */
API.Client.AccountInfo.prototype.oauthproviders;

/**
 * @type {!API.Client.AccountInfoOauthConfig}
 * @export
 */
API.Client.AccountInfo.prototype.oauthconfig;

/**
 * @type {!Array<!string>}
 * @export
 */
API.Client.AccountInfo.prototype.oauthadapters;

/**
 * @type {!API.Client.AccountInfoLimits}
 * @export
 */
API.Client.AccountInfo.prototype.limits;

/**
 * @type {!string}
 * @export
 */
API.Client.AccountInfo.prototype.language;

/**
 * @type {!API.Client.AccountInfoCountryCurrencies}
 * @export
 */
API.Client.AccountInfo.prototype.countryCurrencies;

/**
 * @type {!boolean}
 * @export
 */
API.Client.AccountInfo.prototype.enableLocales;

/**
 * @type {!boolean}
 * @export
 */
API.Client.AccountInfo.prototype.enableCurrencies;

/**
 * @type {!string}
 * @export
 */
API.Client.AccountInfo.prototype.gravatar;

